### PR TITLE
citra-qt: Use monospace font on Disassembler and ARM Registers

### PR DESCRIPTION
**Pull request by Kloen**
Picked up #156 if order to finish #67. If any other debugger tool should use it, just say it and I'll add it.

### CODE REVIEW

File: src/citra_qt/main.cpp [hunk 1]
```diff
@@ -58,6 +58,10 @@ GMainWindow::GMainWindow() : emu_thread(nullptr)
     ui.setupUi(this);
     statusBar()->hide();
 
+    QSettings settings;
+    if (settings.value("debugger_font").isNull())
+        settings.setValue("debugger_font", QVariant("Courier"));
```
Comment (archshift) [inline src/citra_qt/main.cpp]: Courier isn't available on all platforms.

**Comment by yuriks** [comment]
We have a function on util/util.cpp which I created to avoid duplicating monospace font policy through the codebase, please use that function and then if desired implement the option to allow the user to choose a font based using that too.

**Comment by Kloen** [comment]
@yuriks wops, didn't saw it, definitely Notepad++ it's not as good as VS to help -.-'

**Comment by yuriks** [comment]
LGTM (please squash)
I'd like to see the font configuration thing you before too, but you can probably do that in a different PR.

**Comment by Kloen** [comment]
Squashed. 

I'd like to do the font configuration if I could right now. But QT doesn't like vs2015 at all -.-' so I'll look into it once I can build citra on my PC, or on my freetime at work.

**Comment by bunnei** [comment]
LGTM

**Pull request merged**